src/shared/common/Button.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
```

Comment:
Can you add a `// @flow` on the first line of the file to make sure flow looks at it?